rasa/shared/core/trackers.py

```diff
@@ -271,9 +271,22 @@ def change_form_to(self, form_name: Text) -> None:
         )
         self.change_loop_to(form_name)
 
+    def interrupt_loop(self, is_interrupted: bool) -> None:
+        """Interrupt loop and mark that we entered an unhappy path in the conversation.
+        Args:
+            is_interrupted: `True` if the loop was run after an unhappy path.
+        """
+        self.active_loop[LOOP_INTERRUPTED] = is_interrupted
+
     def set_form_validation(self, validate: bool) -> None:
-        """Toggle form validation"""
```

Comment:
Can we keep the docstring?

Reply:
I removed it since it just calls the other method